manual/source/classes/contributed_definitions/em-structure.rst

```diff
@@ -0,0 +1,134 @@
+Partner consortia in the German National Research Data Infrastructure are here e.g.
+NFDI-BioImage, NFDI-Microbiota, NFDI4Health, and e.g. NFDI-Neuro.
+
+Electron microscopes are functionally very customizable tools: Examples include multi-signal/-modal analyses which are frequently realized as on-the-fly computational analyses, regularly switching between GUI-based instrument control, computational steps, and more and more using high-throughput stream-based processing. Also artificial intelligence methods are increasingly used and become closer interconnected with classical modes of controlling the instrument and perform data processing. A challenge in electron microscopy is that these steps are often executed within commercial integrated control and analysis software. This makes it difficult to keep track of workflows in a technology-partner-agnostic,
+i.e. interdisciplinary manner.
+
+.. _EmAppDef:
+
+Application Definitions
+#######################
+
+We acknowledge that it can be difficult to agree on a single application definition which is generally enough applicable yet not unnecessary complex and useful for application across a variety of instruments, technology partners, and instrument use cases. In what follows, the proposal conceptualizes first the basic components of an electron microscope and the usual workflow how an electron microscope is used for collecting data with detectors via probing radiation-specimen-matter interaction mechanisms.
+
+In summary, scientists place a specimen/sample into the microscope, calibrate the instrument, take measurements, may perform experiments, prepare their specimens with a focused ion beam, calibrate again, and take other measurements, before their session on the instrument ends. In between virtually all of these steps data are collected and stream in from different detectors probing different physical mechanisms of the interaction between electrons or other types of radiation with the specimen.
+
+A microscope session ends with the scientist removing the specimen from the instrument or parking it so that the next user can start a session. Occasionally, service technicians perform calibrations and maintenance which also can be described as session on the microscope. Next, we wrote base classes to describe these steps and events.
+
+    :ref:`NXem`:
+        A general application definition which explores the possibilities of electron microscopes.
+
+.. _EmBC:
+
+Base Classes
+############
+
+The following base classes are proposed to support modularizing the storage of pieces of information:
+
+    :ref:`NXaberration_model`, :ref:`NXaberration_model_ceos`, :ref:`NXaberration_model_nion`, :ref:`NXaberration`:
+        Base classes to describe procedures and values for the calibration of aberrations based on either CEOS or Nion.
+
+    :ref:`NXaperture_em`:
+        A base class to describe an aperture.
```

Comment:
Agree, NXslit and or NXaperture would match:
However, there is an inconsistence currently:
https://manual.nexusformat.org/classes/base_classes/NXaperture.html#nxaperture
vs
https://manual.nexusformat.org/classes/base_classes/NXslit.html#nxslit
an EM aperture maybe qualifies for be more than simple so NXaperture as a compromise to use but that is deprecated ...

Reply:
Unless that inconsistence is resolved let's keep NXaperture_em, although if it is really an NXaperture only for em, then we should revert the symbol for the base class i.e NXem_aperture to although I have to say we have a constant struggle with the ordering of the field-specific "prefix" like "em" and the symbol for the concept specifier here "aperture", I agree though that the em specific content could equally go into NXem and then one would just say (NXaperture) or (NXslit) (in yaml notation) but it is really the fact that aperture is deprecated because of which I am hesitant specifically because either way fact is we have aperture (several of them in EM) and we want need to describe them and these apertures have similarities with other components called aperture useful not only for EM

Reply:
Also noted as in an issue in #1231.  I think  "This group is deprecated, use NXslit instead" overrides what appears to be left over text "A simple slit. For more complex geometries, NXaperture should be used." in NXslit.  Therefore I think my question remains.  What is missing from NXslit that requires a new NXaperture_em base class?  Or maybe better stated, could we pull enough stuff into NXslit from NXaperture to make NXaperture_em unneeded?

Reply:
Argument from @sanbrock that convinced me: making this base class avoids a lot of copy-pasting between many application definitions.  Resolving.  Thanks!